Sprintfinity3902/HudMenu/InventoryHud.cs

```diff
@@ -116,10 +116,47 @@ public void RemoveItemInInventory(IPlayer.SelectableWeapons weapon)
                 availableItems.Remove(weapon);
         }
 
+        /*public void MoveSelectorRight()
```

Comment:
If these comments are not ever going to be used again, can we just get rid of them?